wandb/internal/sender.py

```diff
@@ -212,6 +283,7 @@ def handle_stats(self, data):
         self._flatten(row)
         row["_wandb"] = True
         row["_timestamp"] = now
+        # TODO: run has a start_time, as well as history and settings :(
         row["_runtime"] = int(now - self._settings._start_time)
```

Comment:
This is bad and needs to be fixed.  I think `settings._start_time` will be the first time wandb.init was called, so this falls apart with with `reinit`.  I think we should move to using `start_time` from the run obj, but I don't have access to run in `handle_stats`...

Reply:
I think this is actually fine, we get a new self._settings._start_time with each wandb.init() call